docs/developer-docs/integrations/sns/get-sns/local-testing.md

```diff
@@ -134,54 +241,54 @@ If the proposal is rejected, the dapp canisters' controllers are automatically s
 back to the developer principals that you
 have defined in the [initialization file](preparation.md).
```

Comment:
```suggestion
have defined in the [initialization](preparation.md).
```

(actually I think most of those params are now defined in the NNS proposal, so I suggest we are more generic and just say "in the initialization")